storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7343,9 +7340,9 @@ static int check_privilege(
                                          is not granted */
 {
   if (!has_privilege(granted_priv, required, target_database, target_table)) {
-    msg("xtrabackup: %s: missing required privilege %s on %s.%s\n",
-        (error == PRIVILEGE_ERROR ? "Error" : "Warning"), required,
-        target_database, target_table);
+    xb::error() << (error == PRIVILEGE_ERROR ? "Error" : "Warning")
+                << " missing required privilege " << required << " on "
+                << target_database << "." << target_table;
```

Comment:
Here we will need and in for each message, otherwise in case of warning we will print `[error] [xtrabackup] Warning ...`

Reply:
right, fixed it